web/src/components/layout/flex/styles.tsx

```diff
@@ -10,6 +10,7 @@ export default createUseStyles(() => {
     flex: {
       display: 'flex',
       flex: 1,
+      width: ({ width }) => width,
```

Comment:
let's use 100% as default here? 
`width: ({ width }) => width || '100%',`

Reply:
I was afraid to do that but that would be even better :D

Reply:
isn't Flex used in a lot of places? we could break something

Reply:
Most likely it will not as wrapper parent exist for each Flex component